src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.icon {
+  width: 40px;
+  height: 40px;
+
+}
+
+/* .nav {
+  display: flex;
+} */
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.nav__list__item {
+  margin: 0 20px 0 0;
+}
+
+.nav__list__item:last-child {
+  margin-right: 0;
+}
```

Comment:
You can finde how it work using query in google: `:not() with :last-child`
```suggestion
.nav__item:not(last-child) {
  margin-right: 20px;
}
```
<img width="1260" alt="image" src="https://user-images.githubusercontent.com/99652389/178925800-6fe787a5-1f1a-405b-8fb6-7671abe1faf5.png">